cypress/features/regression/buttonToggleClassic.feature

```diff
@@ -0,0 +1,48 @@
+@ignore
```

Comment:
you can't ignore all feature only individual TC

Reply:
The tag does actually ignore the complete feature. I'll leave the `@ignore` tag in until we find a solution to round up the width to an integer.

Reply:
It used to work with these long decimals and some integers until Chrome updated to v80. Now none of them work. Rather than fail every run, it may as well be ignored until we can make it less brittle. I previously added a comment about why it is ignored stating "@ignore until solution is applied to round sizes as integers as Chrome v80 has amended pixel sizes".